### DISCUSSION TITLE
how to E sign a pdf file

### DISCUSSION THREAD

**Discussion by ben135vak** [MatthiasValvekens/pyHanko#300] [Q&A]
Hey.
we have a digital certificate and ive downloaded it to a pfx certificate.
and i want to create a python script that E sign a password protected pdf file using the certificate (or if there is a better way i would love to know).\
if anyone have any insights i would love to know.
thank you :)

**Comment by MatthiasValvekens** [comment]
Hi,

This is a very generic question. Please review [the documentation](https://pyhanko.readthedocs.io/en/latest/lib-guide/index.html) first. If you still have questions afterwards, you are of course more than welcome to ask (targeted!) follow-up questions in another post.
